index.js

```diff
@@ -1,17 +1,78 @@
 class SortedList {
-  constructor() {}
 
-  add(item) {}
+  //Iteration 1: constructor()
+  constructor(items, length) {
+       this.items = [];
+      this.length = 0;     
+  }
 
-  get(pos) {}
+  // Iteration 2: add(item)
+  add(item) {
+    if (typeof item != 'undefined') {
```

Comment:
for this method, there was no usecase where an item would be falsy.
But if you want to prepare for this, instead of wrapping the usable / correct code inside a big if statement, you can prepare beforehand like this:
```js
if (!item) //this handles any falsy values,
``` 
and you throw an error there, and then do your logic then